spec/lib/webauthn/security_key_authentication_service_spec.rb

```diff
@@ -158,6 +162,36 @@
     end
   end
 
+  context 'for windows hello (alg -257)' do
+    ##
+    # These are sourced from an actual login using the UserSecurityKey credential
+    # defined in this spec, generated via a local discourse.
```

Comment:
I think it'll be helpful to document how it can be generated locally so that another developer can benefit from it in the future.